tools/keymaster/keymaster.py

```diff
@@ -15,6 +15,14 @@
 import sys 
 import time
 
+# Set up prometheus metrics 
+metrics = {
+    "wallet_balance": Gauge("ethereum_wallet_balance", "ETH Wallet Balance", ["role", "home", "address", "network"]),
+    "transaction_count": Gauge("ethereum_transaction_count", "ETH Wallet Balance", ["role", "home", "address", "network"]),
+    "block_number": Gauge("ethereum_block_height", "Block Height", ["network"]),
+    "failed_tx_count": Counter("keymaster_failed_tx_count", "Number of Failed Keymaster Top-Ups", ["network", "to", "error"])
```

Comment:
Why using a Counter here and not a Gauge like you did for transaction_count?

Reply:
Since transaction_count is being sourced from the (load-balanced) RPC, using a Gauge here avoids errors that could occur given the eventually-consistent nature of RPC calls.